docs/cookbooks/common_workflows.md

```diff
@@ -0,0 +1,97 @@
+# Common bigquery-etl workflows
+
+This is a quick guide of how to perform common workflows in bigquery-etl using the `bqetl` CLI.
+
+For any workflow, the bigquery-etl repositiory needs to be locally available, for example by cloning the repository, and the `bqetl` CLI needs to be installed by running `./bqetl bootstrap`.
+
+## Adding a new scheduled query
+
+1. Run `./bqetl query create <dataset>.<table>`
+    * Specifiy the desired destination dataset and table name for `<dataset>.<table>`
+    * Directories and files are generated automatically 
+1. Open `query.sql` file that has been created in `sql/moz-fx-data-shared-prod/<dataset>/<table>/` to write the query
+1. Run `./bqetl query schema update <dataset>.<table>` to generate the `schema.yaml` file
+    * Optionally add column descriptions to `schema.yaml`
```

Comment:
The tutorial does not currently have this information, which was just added.

Reply:
I do think it's more likely that we'll keep this short form up to date. Updating the long-form involves wordsmithing and explaining context that requires more effort than updating a bullet point. Given that the long-form version is told in a specific context for creating a specific dataset, I think there's some challenge there in terms of what it means to keep it "up to date" since we'd be talking about actions that were not actually performed for the case in question. It may be reasonable to preface that long-form piece with a statement that it's a snapshot in time of the workflow as of January 2021.

Reply:
Currently, adding a schema is not strictly required (we only have it for clients_daily) so the steps in the tutorial are still very much valid as described. I do think that in the long run we might want to migrate over to schemas, that's why I included them here. Once that becomes the standard it'd make sense to update the tutorial.

Reply:
Ok, got it. Maybe best to add a comment here that this is not yet required, but recommended. Sounds like we can update the tutorial later.